utils/load_utils.py

```diff
@@ -0,0 +1,377 @@
+"""
+Loading data helper functions
+"""
+
+import pathlib
+import pandas as pd
+
+from .data_utils import process_tissue_ccle_column, process_missing_tissues
+
+# Set constants
+# Note the dictionary key is both the dataset and also the directory
+FILENAMES = {
+    "ccle": {"viability": "CCLE_NP24.2009_Drug_data_2015.02.24.csv"},
+    "depmap": {
+        "viability": "CRISPRGeneDependency.csv",
+        "celllines": "Model.csv",
+        "genes": "depmap_gene_dictionary.tsv",
+    },
+    "prism": {
+        "primary_screen": {
+            "viability": "primary-screen-replicate-collapsed-logfold-change.csv",
+            "celllines": "primary-screen-cell-line-info.csv",
+            "treatments": "primary-screen-replicate-collapsed-treatment-info.csv",
+        },
+        "secondary_screen": {
+            "viability": "secondary-screen-replicate-collapsed-logfold-change.csv",
+            "celllines": "secondary-screen-cell-line-info.csv",
+            "treatments": "secondary-screen-replicate-collapsed-treatment-info.csv",
+        },
+    },
+    "nci60": {"viability": "DOSERESP.csv", "treatments": "chemnames_Aug2013.zip"},
+}
+
+CCLE_MISSING_TISSUE_NAMES = {
+    "M059J": "GLIAL",
+    "SF8657": "FIBROBLAST",
+    "SNUC2B": "LARGE_INTESTINE",
+}
+
+DEPMAP_MISSING_TISSUE_NAMES = {
+    "NCC-LMS1-C1": "BONE",
+    "NCC-MPNST2-C1": "MPNST",
+    "A375_RPMI": "SKIN",
+    "RVH421_RPMI": "SKIN",
+}
+
+
+def load_ccle(
+    top_dir, data_dir="ccle/data", process_tissues=False, only_get_cells=False
+):
+    """
+    Load Cancer Cell Line Encyclopedia (CCLE) viability data
+
+    Arguments:
+    ----------
+    top_dir : str
+        Where the repository top level directory is in relation to your current wd
+    data_dir : str, default "ccle/data"
+        The intermediate level folder where data are stored
+    process_tissues : boolean, default False
+        Whether or not to extract and clean tissue from cell line identifiers
+    only_get_cells : boolean, default False
+        Decision to load only columns (without duplicates) of cell line info
+
+    Returns:
+    --------
+        The CCLE viability information per cell line : pandas.DataFrame
+    """
+    # Only get cells works if process_tissues=True
+    if only_get_cells:
+        assert process_tissues, "process_tissues must be True if only_get_cells is True"
```

Comment:
Consider raising an exception based on the multiple boolean conditions instead of using assert here. Doing this may increase readability, allows for greater flexibility with Exceptions (if desired), and may avoid patterns where asserts can be disabled (Python allows this using the `python -O` flag when running files).
```suggestion
    if only_get_cells and not process_tissues:
        raise Exception("process_tissues must be True if only_get_cells is True")
```

Reply:
nice!